test/string_validator_test.rb

```diff
@@ -0,0 +1,27 @@
+# gem 'minitest'
```

Comment:
You're always asserting that a thing happens. Would liked to have seen some sad path and edge case testing.